script/Ciel/Day 14/14_01B_01_CIEL14B_1E.txt

```diff
@@ -321,122 +346,131 @@ I hate Roa. I'll kill Roa.
 -- Page 234, Offset 39041.
 -- 意識すればするほど順番が変わってしまう。
-The more she tries to make sense of it, the more disorderly she becomes.
+The more she tries to think,
+ the more muddled her mind becomes.
 }
 [sha:629585d7f9e6671f51537bbb39037a2da939690d]{
 -- Page 235, Offset 39042.
 -- なので、彼女は考える事はやめる事にした。
-As such, she decided to stop thinking.
+And so, she decides to stop thinking.
 }
 [sha:07d6175143ebd3db43edda9dd8a821055b33063f]{
 -- Page 235, Offset 39043.
 -- 自分の気持ちを考えたり、彼の気持ちを考えたりするから混乱する。
-Things are chaotic because she continues to think about her own feelings and about him.
+She's confused because she's trying to think about her emotions; about his emotions.
 }
 [sha:1f25ea6d796e25cb43eb9335ac1b596008ef8279]{
 -- Page 235, Offset 39044.
 -- 彼女は原始的に、それ以上に純粋無垢に、<や|・><り|・><た|・><い|・><事|・><だ|・><け|・><を|・><や|・><る|・>獣になった。
-She turns herself into a primitive beast; so primitive, she chooses only to do the things she wants to do.
+So instead, she turns herself into the platonic ideal of a primitive beast――one driven by nothing but its own desires.
 }
 [sha:d7a4591408acfa308d2b5b2f8a8314ccc067fe3e]{
 -- Page 236, Offset 39045.
 -- “待ってて―――<跡形|アトカタ>も残さないから―――！”
-"Just you wait―――I won't leave behind even a single trace―――!"
+"Just you wait――I won't leave so much as a trace―――!"
 }
 [sha:975fe8619e56d338f6ba2f4520abc88af3ae5dcb]{
 -- Page 236, Offset 39046.
 -- 彼女は蘇生を終え、顔をあげた。
-With the resurrection complete, Arcueid raises her head.
+Her resurrection complete, Arcueid raises her head.
 }
 [sha:d411b757119839967a894da0c0da72b79910f4cd]{
 -- Page 236, Offset 39047.
 -- 周囲の情報はすべて読み切っている。
-She reads the information of her entire surroundings.
+She takes in all the information on her surroundings.
 }
 [sha:826b7c0a364e4cd073b8bbac33a84bd332885e6d]{
 -- Page 236, Offset 39048.
 -- 街の形状、逃げた獲物の居場所、隠れて機を狙っている虫の居場所、すべて手に取るように解っている。
-The shape of the city, the location of the runaway prey, the location of the hiding insect waiting for an opportunity; she understands it all in perfect detail.
+The layout of the city,
+ the location of her runaway prey,
+ the location of the insect hiding and waiting for an opportunity――she understands it all with perfect clarity.
 }
 [sha:d1a524878b79417555a48bd33ac923b8e39dc592]{
 -- Page 236, Offset 39049.
 -- 絶好の行楽日和。
-A perfect day for going out.
+It's the perfect day for an outing.
 }
 [sha:b0a1569c42cb0c16e576c08347c6b89400003111]{
 -- Page 236, Offset 39050.
 -- 数秒後の惨劇を思い描き、満を持しての一歩を踏み出す。
-She imagines the tragedy that is about to occur in the coming seconds, and takes her first step in anticipation.
+Picturing the tragedy that it is about to unfold,
+ she takes her first long-awaited step.
 }
 [sha:ff13c8fe9d8bf77bcb45469d32401f51692cd3d5]{
 -- Page 237, Offset 39051.
 -- その、寸前。
-But at that moment.
+But just as she does―――
 }
 [sha:73e599d1a8e5c0c0fce2b4f0214e3f0ef850a5b8]{
 -- Page 237, Offset 39052.
 -- 「ああもう―――ちょっと目が覚めちゃったじゃない」
-"Oh come on―――I only just woke up."
+"Oh come on... I only just woke up."
```

Comment:
Misinterpretation.
She's saying Ciel's shot woke her up.
It's obviously a bit mocking.
>Oh, come on... there you go waking me up a tad.